app/views/sidebar/_question.html.erb

```diff
@@ -24,13 +24,13 @@
       <% end %>
     <% end %>
 
+    <%= render partial: "sidebar/map" %>
     <%= render partial: 'sidebar/notes', locals: { notes: @notes, title: "Related Questions", node: @node } %>
```

Comment:
We are going to need a margin or some other gap between these two partials, sidebar/map and sidebar/notes. They are squished together!

Reply:
@nstjean Have updated the pr :smile: Can you kindly review again? Thanks :v: Here is a ss of the change -  
![Screenshot from 2020-03-09 19-07-15](https://user-images.githubusercontent.com/33183263/76217900-4cd66b80-6239-11ea-866e-cde6ca074a7b.png)